### DISCUSSION TITLE
Are there any papers related to rook?

### DISCUSSION THREAD

**Discussion by fengjiankui121** [rook/rook#10594] [Q&A] [answered]
Are there any papers related to rook?

**Comment by zhucan** [comment, marked as answer]
https://github.com/rook/rook/tree/master/Documentation Is it ok for you?

**Reply by fengjiankui121** [reply]
No, I need an article published in a conference or journal.

**Reply by travisn** [reply]
There are papers related to Ceph core features such as RADOS, but I'm not aware of papers for Rook specifically.
